functions/Invoke-DbaDbDataGenerator.ps1

```diff
@@ -214,12 +214,16 @@ function Invoke-DbaDbDataGenerator {
                                     if ($columnMaskInfo) {
                                         # Generate a new value
                                         try {
+                                            if ($PSBoundParameters.MaxValue -and $columnMaskInfo.SubType -eq 'String' -and $columnMaskInfo.MaxValue -gt $MaxValue) {
+                                                $columnMaskInfo.MaxValue = $MaxValue
+                                            }
+                                            # TODO: $columnobject is not set here, so should this be $columnMaskInfo?
+                                            # TODO: Is SubType correct or should this be Type? In my example config for "CREATE TABLE test (a varchar(20))", Type is varchar, SubType is String
```

Comment:
The DataType should be `$columnMaskInfo.ColumnType` here because it wants the data types here. The data type `varchar` will be converted in the command `Get-DbaRandomizedValue` to a random `String2`